### PR TITLE
SALTO-5482: Allow trexcloud domain in okta login

### PR DESCRIPTION
**Pull request by shir-reifenberg**
This is temporary addition, will remove once all tests are done
More details in linked ticket 

---

_Additional context for reviewer_

---
_Release Notes_: 
None

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/okta-adapter/src/adapter_creator.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ const credentialsFromConfig = (config: Readonly<InstanceElement>): Credentials =
   const { value } = config
   const { baseUrl } = value
   const hostName = new URL(baseUrl).hostname
-  if (!hostName.includes(OKTA)) {
+  if (!hostName.includes(OKTA) && !hostName.includes('trexcloud')) {
```
[the remark this answers is not in the document]
Comment (amitizle) [reply packages/okta-adapter/src/adapter_creator.ts:62]: Can the hostname be explicit in here?
The test `hostname.includes(OKTA)` will also return true on `this-is-not-okta.com`.

[the remark this answers is not in the document]
Comment (shir-reifenberg) [reply packages/okta-adapter/src/adapter_creator.ts:62]: re okta - I don't wanna block this cause there are some types of domains.. (and anyway we have a more explicit check in saas)
re trexcloud - @omrilit what do you suggest? same they can't guarantee to use a specific url

I can add a check to verify it starts with `https://`

@omrilit @amitizle WDYT ?

[the remark this answers is not in the document]
Comment (amitizle) [reply packages/okta-adapter/src/adapter_creator.ts:62]: For sure add a check that `protocol === 'https'`.
To be honest I'd add a list of domains and maybe an env var that can modify this list (not adapter config, but env variable).
How many options are there? There are three different apex domains in [here](https://developer.okta.com/docs/guides/find-your-domain/main/);

- `oktapreview.com`
- `okta.com`
- `okta-emea.com`

If that's the case, I suggest to check that the input is either that or `trexcloud` in a very specific way.

**Review by omrilit — APPROVED** [review]
🏅

**Pull request merged**